.github/workflows/check-archs-consistency.yml

```
@@ -0,0 +1,26 @@
+name: Test consistency between Paper & Fabric
+on: 
+  pull_request:
+    branches:
+      - main
+    paths: 
+      - src/fabric/**
+      - android/src/paper/java/com/facebook/react/viewmanagers/**
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: check-archs-consistency-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js 18
+        uses: actions/setup-node@v2
```

Let's use `v4` script versions.

```suggestion
        uses: actions/checkout@v4
      - name: Use Node.js 18
        uses: actions/setup-node@v4
```

---

:(